packages/rum-core/src/browser/windowOpenObservable.ts

```diff
@@ -0,0 +1,9 @@
+import { instrumentMethod, Observable } from '@datadog/browser-core'
+
+export function createWindowOpenObservable() {
+  const observable = new Observable<void>()
+  instrumentMethod(window, 'open', () => {
+    observable.notify()
+  })
+  return observable
+}
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏ You solution works, but we could improve it to
  - avoid instrumenting multiple time unnecessarily in case `createWindowOpenObservable` is called more than once
  - stop intrumenting when the SDK stops (we'll have to adjust things in startRum)


```suggestion
let windowOpenObservable;

export function createWindowOpenObservable() {
  if (windowOpenObservable) {
    return windowOpenObservable;
  }
  
  const observable = new Observable<void>()
  const { stop: stopInstrumentation } = instrumentMethod(window, 'open', () => {
    observable.notify()
  })
  
  return { 
    observable, 
    stop() {
      stopInstrumentation()
      windowOpenObservable = undefined;
    }
  }
}
```